index.js

```diff
@@ -227,28 +227,28 @@ function loadConfig(program) {
         ...config
       , ...parsedConfig
       , tags: process.env.LOGDNA_TAGS || config.tags
-      , flushTimer: null
       , healthcheckTimer: null
       , rescanTimer: null
       }
 
+      return getos(cb)
+    }
+  , (distro, cb) => {
+      let distroName = `${(distro.dist || distro.os).replace(/Linux| /g, '')}`
+      if (distro.release) {
+        distroName += `/${distro.release}`
+      }
+      config.UserAgent = `${pkg.name}/${pkg.version} ${distroName}`
+
       if (process.env.LOGDNA_PLATFORM) {
-        config.platform = process.env.LOGDNA_PLATFORM
-        config.tags = config.tags ? `${config.tags},${config.platform}` : config.platform
-        if (config.platform.indexOf('k8s') === 0) {
+        const platform = process.env.LOGDNA_PLATFORM
```

Comment:
`config.platform` was being mainly used for `k8s` but since we are gonna remove it, I think, no need for `config.platform` but need to double check where else it gets used

Reply:
Thanks.  I can take it out on the next PR which will be k8s specific.  Lemme know what you find.